app/mailers/enrollment_mailer.rb

```diff
@@ -6,13 +6,17 @@ class EnrollmentMailer < ActionMailer::Base
       :validate_application => 'Votre demande a été validée',
       :review_application => 'Votre demande requiert des modifications',
       :refuse_application => 'Votre demande a été refusée',
-      :update_contacts => 'Contacts modifiés sur signup.api.gouv.fr'
+      :update_contacts => 'Contacts modifiés sur signup.api.gouv.fr',
+      :send_application_feedback => "Nous avons bien reçu votre demande d'accès"
   }
 
-  %i[send_application validate_application review_application refuse_application update_contacts].each do |action|
+  %i[send_application validate_application review_application refuse_application update_contacts send_application_feedback].each do |action|
```

Comment:
feedback make me think of the user giving us feedback rather than the application giving feedback to the user. Maybe we should use something more like "send_application_notification" instead.

Reply:
`sent_application_notification` seemed more explicit

Reply:
ok. Why do you use past form instead of infinitive form like other actions ?